CHANGELOG.md

```diff
@@ -1,5 +1,29 @@
 # Changelog
 
+## v1.0.3
+
+*Jan 20, 2022*
+
+* Improve p2p/peer reactor
```

Comment:
Could you add more information about backports?

Reply:
OK. I will fix it!